cmd/coreinstance/create_core_instance_operator.go

```diff
@@ -343,6 +344,7 @@ func newCmdCreateCoreInstanceOperator(config *cfg.Config, testClientSet kubernet
 	fs.StringVar(&environment, "environment", "", "Calyptia environment name")
 	fs.StringVar(&httpProxy, "http-proxy", "", "http proxy to use on this core instance")
 	fs.StringVar(&httpsProxy, "https-proxy", "", "http proxy to use on this core instance")
+	fs.StringVar(&annotations, "annotations", "", "Custom annotations for pipelines. Format should be 'annotation1=value1,annotation2=value2'")
```

Comment:
Should we build in support for labels now as well as part of this PR for future proofing? It feels like adding a few lines will sort it.

Reply:
let's try not to cram all we can think of into a single PR